packages/react-query/package.json

```diff
@@ -72,21 +72,6 @@
   "bundledDependencies": [
     "tsd"
   ],
-  "publishConfig": {
-    "access": "public",
-    "main": "dist/index.js",
-    "module": "esm/index.mjs",
-    "exports": {
-      ".": {
-        "require": "./dist/index.js",
-        "import": "./esm/index.mjs",
-        "types": "./dist/index.d.ts"
-      },
-      "./package.json": "./package.json"
-    },
-    "types": "dist/index.d.ts",
-    "import": "esm/index.mjs"
-  },
```

Comment:
I thought we need to use dist files, not src/index.ts because I think visualization need to use same files with library users